packages/core/src/observable/index.ts

```diff
@@ -1,4 +1,16 @@
-export * from './observable';
-export * from './types';
-export * from './operators';
+export { isObservable, observable } from './observable';
+export type { inferObservableValue } from './observable';
+export type {
+  MonoTypeOperatorFunction,
+  Observable,
+  Observer,
+  OperatorFunction,
+  Subscribable,
+  SubscriptionLike,
+  TeardownLogic,
+  UnaryFunction,
```

Comment:
this feels and `MonoTypeOperatorFunction` like an internal thing

unsure if they're needed to be exported at all

Reply:
fun note - I don't know what Unary or MonoType really means, i just copied naming from RxJs :D

Reply:
i know computers

![bailey-dog-meme-2](https://github.com/trpc/trpc/assets/459267/e192f4d3-4307-41ee-894f-94823ff6e5ee)